sgl-kernel/csrc/cpu/shm.cpp

```diff
@@ -154,241 +115,19 @@ void reduce_all_buffers(
     char** buffers) {
   switch (scalar_type) {
     case c10::ScalarType::BFloat16:
-      reduce_bf16_buffers(start_elements, num_elements, to_buffer, buffers);
+      reduce_bf16_buffers(start_elements, num_elements, to_buffer, buffers, world_size);
```

Comment:
Added `world_size` argument, which was a global variable. Since the function is moved to another file, adding this argument makes the function more readable.